src/ccs/mod.rs

```diff
@@ -471,12 +469,12 @@ pub mod test {
 
     // generate generators and ro constants
     let _ck = S.commitment_key();
-    let _ro_consts =
-      <<G as Group>::RO as ROTrait<<G as Group>::Base, <G as Group>::Scalar>>::Constants::new();
+    let _ro_consts = <G::RO as ROTrait<G::Base, G::Scalar>>::Constants::new();
+    //      <<G as Group>::RO as ROTrait<<G as Group>::Base, <G as Group>::Scalar>>::Constants::new();
```

Comment:
Is this commented code intended to be left here?

Reply:
Accidents, thanks! Fixed in https://github.com/privacy-scaling-explorations/Nova/pull/40/commits/bc5aa34dd02413b1e2886ec36e0fbcbdbb2424ef